src/mngt/ocf_mngt_core_pool_priv.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright(c) 2012-2018 Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause-Clear
+ */
+
+#ifndef __OCF_MNGT_CORE_POOL_PRIV_H__
+#define __OCF_MNGT_CORE_POOL_PRIV_H__
+
+#include "ocf/ocf.h"
+
+void ocf_mngt_core_pool_init(ocf_ctx_t ctx);
```

Comment:
Do we really need this since core_pool_init/deinit are already declared in ocf_mngt.h?

Reply:
Fixed. Now it's not. As OCF does it automatically, it no longer needs to be part of public API.